src/Android/Avalonia.Android/AvaloniaActivity.cs

```diff
@@ -33,7 +33,7 @@ public object Content
             }
         }
 
-        public override bool DispatchKeyEvent(KeyEvent e) =>
-            View.DispatchKeyEvent(e) ? true : base.DispatchKeyEvent(e);
+        //public override bool DispatchKeyEvent(KeyEvent e) =>
```

Comment:
Don't comment code. If it's not needed anymore, just remove it.

Reply:
No problem, I'v commented it to make the point for reviewers to loot it twice :)